src/Service.Tests/Unittests/SqlQueryExecutorUnitTests.cs

```diff
@@ -279,6 +282,10 @@ Mock<MsSqlQueryExecutor> queryExecutor
                 = new(provider, dbExceptionParser, queryExecutorLogger.Object, httpContextAccessor.Object, handler);
 
             queryExecutor.Setup(x => x.ConnectionStringBuilders).Returns(new Dictionary<string, DbConnectionStringBuilder>());
+
```

Comment:
Can you please add a test here similar to TestHttpContextIsPopulatedWithDbExecutionTime? 

TestHttpContextIsPopulatedWithStatementId for e.g.

Reply:
Created a test that checks whether an info message handler was added to the connection.

I think it would be a bit tricky to test if we are actually getting the statement ID in a unit test, since we don't actually make a request to the data source - I am not sure on how to mock a database response that would trigger the event handler. 

However, I have tested this behavior when running DAB locally.

Let me know if you think the testing is sufficient and whether you have suggestions on this!